guides/source/active_record_multiple_databases.md

```diff
@@ -349,14 +349,15 @@ class ApplicationRecord < ActiveRecord::Base
 end
 ```
 
-Then models can swap connections manually via the `connected_to` API:
+Then models can swap connections manually via the `connected_to` API. If
+using sharding both a `role` and `shard` must be passed:
 
 ```ruby
-ActiveRecord::Base.connected_to(shard: :default) do
+ActiveRecord::Base.connected_to(role: :writing, shard: :default) do
```

Comment:
Should we recommend using `ActiveRecord::Base.writing_role` instead? A little more verbose but it makes changing the role much easier in the future.

Reply:
No that's an internal "feature", apps should always use the symbol.